docs/data_requirements/input_data.md

```diff
@@ -35,29 +37,33 @@ For example:
 
 
 
-## Naming of column headers
+### Naming of column headers and table name
```

Comment:
I still think this is a bit unstructured. How about you move this section up and name it "Naming conventions for data tables and parameters" ?

Reply:
The distinction of column headers and parameter names is not clear to me at first sight and since all other column headers except for the parameter_names are fixed, I suggest to speak of parameter names throughout this section.